design/one-pager-stack-relationship-labels.md

```diff
@@ -0,0 +1,49 @@
+# Stack Parent/Child Relationship Label Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#934](https://github.com/crossplaneio/crossplane/issues/934)
+
+## Problem
+
+There is currently no easy way to get a list of all resources that are owned by a parent resource. We can leverage CRD labels to make this easier. As long as we have a standard format for doing so.
```

Comment:
>  We can leverage CRD labels to make this easier.

Is CRD labels automatically propagated to instances (CRs) ?

Reply:
I don't believe so.

Reply:
CRs are managed by their own controller.  The Stack Manager does not intervene in the creation or updates of those resources.  It would be possible to setup a webhook from the StackManager to add these annotations if we want to pursue this later.  That would be very helpful in identifying Stack parentage. Nice, @turkenh.

Reply:
In the latest update, https://github.com/crossplaneio/crossplane/pull/961/files#diff-4daf64a45270834c13f7f119f2d6aa98R14, make it more clear that we may not need to label/annotate these 'grandchild' resources at all.

By knowing that a Stack owns a particular Kind/CRD, we can assert that **all** resources of that Kind are owned by the CRD owning Stack.